openshift-knative-operator/pkg/monitoring/monitoring_eventing.go

```diff
@@ -18,13 +18,12 @@ func ReconcileMonitoringForEventing(ctx context.Context, api kubernetes.Interfac
 }
```

Comment:
Is it possible to move some `monitoring_kafka.go` in here?

Reply:
Which part you want to be moved here?